src/main/resources/stsjorbsmodResources/localization/eng/JorbsMod-Card-Strings.json

```diff
@@ -263,6 +263,10 @@
     "NAME": "Frostbite",
     "DESCRIPTION": "Deal !D! damage. NL If this kills an enemy, stsjorbsmod:Consume this."
   },
+  "stsjorbsmod:Frustration": {
+    "NAME": "Frustration",
+    "DESCRIPTION": "Retain. NL Deal !D! damage to ALL enemies. NL Whenever this is retained, deal !M! damage to ALL enemies. NL stsjorbsmod:Exert."
```

Comment:
I think the wording that you've done here is closer to how the base game would word it than the version in the spreadsheet, but I think there are planned to be enough "retain / exert" sort of cards and that they read so much better with the spreadsheet version that I'm okay establishing the convention from the spreadsheet for them. `Retain: Deal !M! damage to ALL enemies. NL stsjorbsmod:Exert: Deal !D! damage to ALL enemies.`

(assuming the `stsjorbsmod:Exert:` with the colon on the end actually works in practice; I'm suspicious)